<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bootstrap Studio To Django</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/img/django.svg</normaloff>:/images/img/django.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabs">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="gettingStartedTab">
       <attribute name="title">
        <string>Getting Started</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>869</width>
             <height>627</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>2. Copy the following EXE filename (namely this app itself) to the BSS &gt; Export Settings &gt; Export Script field 
 (see green rectangle below).</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="copyExeFilenameButton">
              <property name="toolTip">
               <string>Copy this EXE's filename to your clipboard.</string>
              </property>
              <property name="text">
               <string>ðŸ“‹</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="exeFilenameLine">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>4. In BSS click Save to exit the settings screen, save your design and click the Export button.</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLineEdit" name="djangoProjectRootLine">
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Enter in or browse for your Django project's root directory (usually where manage.py sits)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>3. Provide the root directory of your Django project:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>1. Decide where you'd like BSS to copy intermediate project files and specify this in BSS &gt; Export Settings 
 (see blue rectangle below).</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../resources.qrc">:/images/img/bss-export-settings-screen.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="djangoProjectBrowseButton">
              <property name="toolTip">
               <string>Copy this EXE's filename to your clipboard.</string>
              </property>
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statusLogTab">
       <attribute name="title">
        <string>Status Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QPushButton" name="clearLogButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="copyLogToClipboardButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Copy To Clipboard ðŸ“‹</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QPlainTextEdit" name="statusLogText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="openIssuesForumButton">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Open the Issues Forum ðŸ”—</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="helpTab">
       <attribute name="title">
        <string>Help</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;BSS files in their root folder will go into the root of the Django project.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(TODO)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="runExporterButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Run Exporter</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QProgressBar" name="exportProgress">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
